src/frontend/src/elements/animations/LetterHightlight.tsx

```diff
@@ -0,0 +1,26 @@
+// import { PropsWithChildren } from "react";
+import "./style.css";
+
+
+interface Props {
```

Comment:
```suggestion
type LetterHighllightProps {
```
Bennenung des Props hautsächlich für die IDEs